ascon-aead/Cargo.toml

```diff
@@ -20,22 +20,23 @@ subtle = { version = "2", default-features = false }
 zeroize = { version = "1.6", optional = true, default-features = false, features = [
     "derive",
 ] }
-ascon = "0.3"
+ascon = "0.4"
 
 [dev-dependencies]
 hex = "0.4"
 aead = { version = "0.5", features = ["alloc"] }
 spectral = { version = "0.6", default-features = false }
 
 [features]
-default = ["alloc", "getrandom", "zeroize"]
+default = ["alloc", "getrandom", "zeroize", "sponge-zeroize"]
```

Comment:
As noted in the other PR, usually, we do not enable `zeroize` by default.

Reply:
Both features, `zeroize` and `sponge-zeroize`, are now removed